hack/test-end-to-end-docker.sh

```diff
@@ -30,7 +30,17 @@ if [[ -z "${BASETMPDIR-}" ]]; then
   sudo rm -rf "${BASETMPDIR}"
   mkdir -p "${BASETMPDIR}"
 fi
-VOLUME_DIR="${BASETMPDIR}/volumes"
+
+# when selinux is enforcing, the volume dir selinux label needs to be
+# svirt_sandbox_file_t
+#
+# TODO: fix the selinux policy to either allow openshift_var_lib_dir_t
+# or to default the volume dir to svirt_sandbox_file_t.
+sudo mkdir -p /var/lib/openshift/openshift.local.volumes
```

Comment:
@bparees when OpenShift is running as a container, it's trying to use nsenter to create the mount in the host's main mount namespace (pid=1). So it's telling the host "mount a tmpfs at /var/lib/openshift/openshift.local.volumes/pods/b85ea4a2-19e4-11e5-a518-001c422dcd49/volumes/kubernetes.io~secret/default-token-43tbh" but that directory actually exists on the host at /tmp/openshift-e2e/containerized/volumes/pods/b85ea4a2-19e4-11e5-a518-001c422dcd49/volumes/kubernetes.io~secret/default-token-43tbh